README.md

```diff
@@ -1 +1,195 @@
-# Capstone
\ No newline at end of file
+# Capstone
+# Fitness-capstone
+
+<a name="readme-top"></a>
+
+
+<div align="center">
+ 
+
+  
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio: Mobile setup] <a name="about-project"></a>
+
+
+**[Portfolio: Mobile setup]** is a project for week 1.
```

Comment:
- [ Optional ] I think it would be better if this section of the readme file referred to the title of the `HTML - JavaScript` Capstone project` instead of the `Portfolio project`.